src/resources/ble_esp32.cc

```diff
@@ -1820,6 +1820,12 @@ PRIMITIVE(write_value) {
   return Smi::from(with_response ? 1 : 0);
 }
 
+PRIMITIVE(handle) {
+  ARGS(Resource, resource)
+  auto element = reinterpret_cast<BleReadWriteElement*>(resource);
```

Comment:
The reinterpret_cast is horrible, but that's the pattern that is used throughout this file.
Needs to get fixed separately.